libcaf_io/caf/io/network/interfaces.hpp

```diff
@@ -20,6 +20,7 @@
 namespace caf::io::network {
 
 // {protocol => address}
+/// @memberof std::map
```

Comment:
This has been added to indicate that `std::map` is not a member of `caf::io::network`. It confuses the typedef and throws an warning otherwise.